### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by jp26jp** [j26design/SEO-for-Events-w-Google-Rich-Cards#1]
> It appears the SiteMap is not being recognized by Google - I am trying to use the Google Data Highlighter and it cannot see my web pages other than the home page.   Says I don’t have a sitemap.
> 
> The Rich Cards that I did using your widget are showing up in the STRUCTURED DATA in GOOGLE SEARCH CONSOLE but not as Rich Cards “Events”
1. The sitemap doesn't have anything to do with whether Google can highlight your webpages.
2. Don't use the highlighter if you are using the schema widgets because that's going to create duplicate data.
3. I need the URL to be able to do anything.

**Comment by jsteveross** [comment]
10-4 on the tagging.  However, the rich cards issue remains - what to do?

**Comment by jp26jp** [comment]
I need the URL

**Comment by jsteveross** [comment]
https://www.centaurtheatre.com

**Comment by jp26jp** [comment]
With events, the price and URL where tickets can be purchased is going to be pretty important. Was there any reason why you hadn't/didn't want to add those?

**Comment by jsteveross** [comment]
The prices vary by day / time and age groups.  I can provide a url for purchasing tickets however.

**Comment by jsteveross** [comment]
just uploaded with price and link to ticket url

**Comment by jp26jp** [comment]
Turn the price off, there's a bug in it that I'll fix asap. Once you do that, have google recrawl your website.

**Comment by jsteveross** [comment]
cool
thanks John
Steve

**Comment by jp26jp** [comment]
Okay so there's actually a different type of markup that is better for the home page. I've almost finished the widget but how many events are you wanting to list from the home page?

**Comment by jsteveross** [comment]
i can set them on their individual show pages 
no need for them to be on home page - was following what i believed to be the appropriate location for the widgets
on average - a professional theatre playhouse in N America has from 3 to 8 shows per season - beyond the 8 is rare - we have 6 mainstage productions
hope this helps

**Comment by jp26jp** [comment]
No you'll keep the one that you have on the individual pages but then you'll use a different one (which I'll send out today) on the home page. The one I'm sending out today is made specifically for listing events. The markup is different and will help with enabling rich cards.

Also, if you wanted to share access to the webmaster property, my google email is john@j-26.com. The authority level of Centaur Theatre is high so it shouldn't have a problem display these rich cards

**Comment by jp26jp** [comment]
Do you have [Master Pages (Premium)](https://www.j-26.com/seo/master-boosters/master-page-premium/)? That would be very helpful in informing Google that your website is the official website of Centaur Theatre

**Comment by jsteveross** [comment]
I did not put the event rich card widget on each individual show page. I have all six on homepage.
So i fix that.
I certainly am open to your help - where/what acess (googlecanalytics?)

**Comment by jsteveross** [comment]
I believe i do

**Comment by jp26jp** [comment]
Move them to the individual pages. You got the last update sent via email, right?

Under Google Webmaster, if you click on 'Manage Property' then you can add a user to the property.

**Comment by jsteveross** [comment]
Will do
Did not get update

**Comment by jp26jp** [comment]
Check your spam folder. Mailchimp is telling me it was sent.

**Comment by jsteveross** [comment]
You’ve been added to analytics.
I’ve checked my spam and junk - nothing from you unfortunately.

**Comment by jp26jp** [comment]
I actually have no idea how you received any of the downloads because in Ecwid, it says that you saved your email with 2 m's in .com haha. I just had it resent.

**Comment by jsteveross** [comment]
Be patient - I have to wear way too many hats for this company - doing the best I can - was never meant to do all of this - learning as I go.  An occasional typo is excusable.  ;-)


I just put all the rich cards on the individual pages - but they were there not updated since you last instructions.  

I wil now need to download this rich card update and will read the instructions.

**Comment by jsteveross** [comment]
Just to recap and since I had the wrong email and perhaps not all updates:

On MASTER page:
Google Analytics V2.0.0 Options

On Homepage:
Sitemap Uploader V1.1.0 Options
SEO for Webpages V6.2.0 Options
Local SEO - Entertainment Business V2.1.0 Options

Showpages:
SEO for Webpages V6.2.0 Options
Event Google Rich Card V1.0.1 Options


I did the update and published - I will request another crawl from Google and will wait a few days and keep you posted. 
Thanks for the assistance.

**Comment by jsteveross** [comment]
I’m still getting this from Google on the Rich Cards

**Comment by jp26jp** [comment]
The Master Page should have the Master Page (Premium) widget. Everything else is good though.

What is this?

**Comment by jsteveross** [comment]
Was a screen capture of Googles notice:

Webmaster tools - under Rich Cards:

“We did not find any structured data for rich cards on your website.”

Will do Master Page (Premium) on home page this eve.  Hope this solves the problem.

**Comment by jp26jp** [comment]
No this wont solve the problem but it will help. You're seeming to have an indexing issue so you should use the sitemap feature that comes with Master Page (Premium).

Another thing I noticed is that your URLs are not redirecting to the URL with or without www. You should specify that you want www or non-www with htaccess. [You can learn how to do that here](https://www.j-26.com/blog/how-to-redirect-to-or-away-from-www/).

You also have quite a few broken links. Those need 301 redirects to valid pages. [You can learn how to do that here](https://www.j-26.com/blog/how-to-create-a-seo-friendly-redirect/)

**Comment by jp26jp** [comment]
Master Page (Premium) needs to go on the Master Page.

**Comment by jsteveross** [comment]
I’m removing all of the widgets - some say http://www.centaurtheatre.com <http://www.centaurtheatre.com/> and others have the secured version.  Google seems to want for us to choose between www and non-www, so I choose to go without the www in preferences.
I did a 301 and had the www subdomain removed.
All should point to https://centaurtheatre.com <https://centaurtheatre.com/>
Uploading entire site clean.
Then will re-install anew starting with PREMIUM.  

I didn’t know I had to register every possible version of our website: http://www <http://www/>  http://centautheatre.com <http://centautheatre.com/> https://www <https://www/>.  https://centaurtheatre.com <https://centaurtheatre.com/>
So I did that as well - but now trying to figure out how to tell Google I want everything to report and reflect only https://centaurtheatre.com <https://centaurtheatre.com/>

**Comment by jp26jp** [comment]
I just noticed another issue. You can't use double quotes (") in the descriptions or titles.

**Comment by jsteveross** [comment]
The double quotes was on one of the 9 shows we have now up.  Fixed.  I believe that we are now pointing all to https://centaurtheatre.com following your instructions.
When I go to Google Search Console for https://centaurtheatre.com I am getting results under Search Appearance -> Structured Data for our Show Pages and for out Box Office page (called Tickets).
However, for Rich Cards, Google reports "We did not find any structured data for rich cards on your website."

**Comment by jp26jp** [comment]
Okay, that's okay for right now because it looks like Google had been indexing more of the www website. If you do a search for centaurtheatre, most of the sitelinks go to www. I saw that the crawler was run today so hopefully we should be seeing some results in the next few days.

**Comment by jsteveross** [comment]
Thanks.
I’m so tempted to use their tagging option but waiting on our rich cards to show up.
We are competing with a few other theaters in this market and need to have a stronger presence on google..  Sitselinks beneath our name to our shows with dates is the objective.  Ideally with a show visual.  That would be awesome.
Thanks for your help again.

**Comment by jsteveross** [comment]
Here's a different problem I have - nothing to do with your widgets.  I'm trying to put this on our Contact Page.  It's a widget for a Parking Finder service.  You supply your Longitude and Lattitude and it shows parking prices for the area.  It's something out audience requests - but the company in NYC is only providing this code and when I try to put on MUSE - it just never shows.  They said it should be put in after the JQUERY code.  But muse does not allow for this insertion point.  And they also said there might be something not going well because of the https of our site.  If you have a sec or any interest in creating something that inserts something other than the HTML insertion that Muse offers.  Think this would be good.

<div id="find_parking_954104820481" class="bp_widget_bttn_red_large"></div>
<script language="javascript" src="http://bestparking.com/js/bp_widget_lightbox.js"></script>
<script language="JavaScript"></script>

**Comment by jsteveross** [comment]
<div id="find_parking_954104820481" class="bp_widget_bttn_red_large"></div>
<script language="javascript" src="http://bestparking.com/js/bp_widget_lightbox.js"></script>
<script language="JavaScript"></script>

**Comment by jsteveross** [comment]
<div id="find_parking_954104820481" class="bp_widget_bttn_red_large"></div>
<script language="javascript" src="http://bestparking.com/js/bp_widget_lightbox.js"></script>
<script language="JavaScript"></script>

**Comment by jp26jp** [comment]
I just did a search to see what was indexed and it looks like 95% of the site that was indexed was www so it could take up to a week for anything to show. You are obviously more than welcome to use their tagging feature, you just can't use both. That's all that I was saying. I would recommend leaving things the way they are for at least a week because Google isn't going to like if things keep getting switched around.

If you do a search by typing in site:https://centaurtheatre.com, you'll see what Google had indexed. I'm sending out an update today that's for your home page and that should help.

Yeah I'd love to help, just send me the code!

**Comment by jsteveross** [comment]
<div id="find_parking_954104820481" class="bp_widget_bttn_red_large"></div>
<script language="javascript" src="http://bestparking.com/js/bp_widget_lightbox.js"></script>
<script language="JavaScript"></script>

**Comment by jsteveross** [comment]
....// <div id="find_parking_954104820481" class="bp_widget_bttn_red_large"></div>
<script language="javascript" src="http://bestparking.com/js/bp_widget_lightbox.js"></script>
<script language="JavaScript"></script>
....///

**Comment by jsteveross** [comment]
sending by email the code

**Comment by jp26jp** [comment]
Sounds good.

It looks like a lot more of the markup is getting picked up now. Make sure you change the widgets out to the ones that were just released.

Also, make sure you're redirecting the missing pages.

**Comment by jsteveross** [comment]
Will do over nite

Looking forward to implementing and seeing results - our first show opens this week, perfect timing.

Will learn to redirect those pages - we remove things regularly - will get in the habit.

Are you going to be doing more Muse Jam sessions?  Let me know - I'm a fan.

Thanks for all

Steve

**Comment by jp26jp** [comment]
Sounds good. Yeah me too! Start by only adding one of the single events to one of the event pages and then verify the markup. If it comes back as valid, then add it to the rest. I'm sure it will, I just want to make sure there aren't any bugs before you add it to more pages.

Once everything has been added, have Google recrawl the site.

Removing things regularly without redirecting them will definitely hurt the site. So if you aren't always redirecting the pages, then it would be better (in terms of performance) to keep the page live.

There aren't any plans to have me on but I might recommend it to dani after we tackle this webmaster issue. I'm sure a lot of users don't know about most of this process.
